src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,66 @@
-export const Pagination = () => {};
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const currentPageMinus = currentPage === 1 ? currentPage : currentPage - 1;
+  const currentPagePlus = currentPage === total ? currentPage : currentPage + 1;
+  const isFirstPage = currentPage === 1;
+  const isLastPage = currentPage === total;
+
+  return (
+    <>
+      <ul className="pagination">
+        <li className={`page-item ${isFirstPage && 'disabled'}`}>
+          <Link
+            onClick={() => onPageChange(currentPage - 1)}
+            to={`?page=${currentPageMinus}&perPage=${perPage}`}
+            data-cy="prevLink"
+            className="page-link"
+            aria-disabled={isFirstPage ? 'true' : 'false'}
```

Comment:
```suggestion
            aria-disabled={isFirstPage}
```
It's already true or false